### PR TITLE
Add PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by JesseStutler**
Add pull request template, allowing contributors to describe PRs in detail, and allowing reviewers to quickly filter and know what needs to be fixed in the PRs

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#### What type of PR is this?
+<!--
+Add one of the following kinds:
+/kind bug
+/kind cleanup
+/kind documentation
+/kind feature
+/kind failing-test
+
+Optionally add one or more of the following kinds if applicable:
```
Comment (Monokaix) [inline .github/PULL_REQUEST_TEMPLATE.md:10]: Why add this option?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+/area deploy
+/area documentation
+/area performance
+/area test
+-->
+
+#### What this PR does / why we need it:
+
+#### Which issue(s) this PR fixes:
+<!--
```
Comment (Monokaix) [inline .github/PULL_REQUEST_TEMPLATE.md:26]: I think we can add the notice that do not add `Fixes #` if the issue has associated with multiple PRs.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (JesseStutler) [reply .github/PULL_REQUEST_TEMPLATE.md]: updated

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (JesseStutler) [reply .github/PULL_REQUEST_TEMPLATE.md:26]: updated

**Comment by JesseStutler** [comment]
Please take a look @Monokaix

**Pull request merged**